srcpkgs/hunspell-pt_PT-preao/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'hunspell-pt_PT-preao'
+pkgname=hunspell-pt_PT-preao
+version=20201029
+revision=1
+wrksrc=${pkgname}-${version}
+depends="hunspell"
+short_desc="Portuguese dictionary for hunspell (prior 1990 Orthographic Agreement)"
+maintainer="Luis Henriques <henrix@camandro.org>"
+license="GPL-2.0-only, LGPL-2.1-only, MPL-1.1"
+homepage="https://natura.di.uminho.pt/wiki/doku.php?id=dicionarios:main"
+distfiles="https://natura.di.uminho.pt/download/sources/Dictionaries/hunspell/${pkgname}-${version}.tar.gz"
+checksum=9427f1b21b9a3e8126fce62d008e6215b0b631498232ae567859d363351fbac4
+
+do_install() {
+	vinstall ${wrksrc}/pt_PT-preao.aff 644 usr/share/hunspell
```

Comment:
I don't think you should be using `${wrksrc}` here, templates are run inside the wrksrc by default.